### DISCUSSION TITLE
Textual lib

### DISCUSSION THREAD

**Discussion by maelp** [darrenburns/posting#29] [General]
The UX is very cool! Are they using only standard Textualize widgets, or you made some custom for this app (auto-complete, jump to position, etc), and in this case consider distributing them in Textualize or in another lib?

**Comment by newtoallofthis123** [comment]
Yeah I was just looking at the components!
Very well designed indeed

**Comment by darrenburns** [comment]
The auto-complete is https://github.com/darrenburns/textual-autocomplete - although Posting is using an alpha version of it `3.0.0a*`, not the main branch. 

The jump mode stuff is built into Posting directly. I may look at extracting it but it's quite specific to the layout of Posting and the interface isn't clean for external use yet. Extracting it would be quite a bit of work to generalise it for all apps.
